notebooks/L1_working_with_age_ensembles.ipynb

```diff
@@ -25,9 +25,10 @@
     "\n",
```

Comment:
<p>chron data based ensembles&nbsp;---&gt; ensembles based on ChronData tables</p><p>paleo data based ensembles --&gt;  ensembles based on PaleoData tables</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LinkedEarth/PyleoTutorials/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Addressed in [b83a608](https://github.com/LinkedEarth/PyleoTutorials/pull/39/commits/b83a60874afed075ce7b2dcc49b71af93035f8ad)